content/cucumber/api.md

```diff
@@ -531,9 +531,19 @@ Or changing your JUnit runner class:
 @CucumberOptions(tags = "@smoke and @fast")
 public class RunCucumberTest {}
 ```
-
 {{% /block %}}
 
+## Ignoring a subset of scenarios
+
+You can tell Cucumber to ignore scenarios with a particular tags:
+
+Using JUnit runner class:
```

Comment:
2) Please mark this line as "java" text, so it only shows up in the Java tab: 
"Using JUnit runner class:" ->
"{{% block "javascript" %}}Using JUnit runner class:{{% /block %}}"

Fyi: You can either build the project yourself locally (check the README for how to do so) and check that any text you add doesn't show up in the wrong place for other language tabs. Alternatively, you can also find a preview of your PR under "Show all checks" -> "Details" to see how it looks.